roles/lib_openshift/src/doc/certificate_authority

```diff
@@ -0,0 +1,141 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oadm_ca
+short_description: Module to manage openshift certificate authority
+description:
+  - Wrapper around the openshift `oc adm ca` command.
+options:
+  state:
```

Comment:
@detiber, it really depends on what it means to exist.  The instance that we use primarily is to generate a signed certificate. In that instance, the exists verifies that the hostnames are in the certificate.  It probably should be broken out into their own modules.